rnd/autogpt_server/autogpt_server/blocks/basic.py

```diff
@@ -174,17 +180,70 @@ def run(self, input_data: InputOutputBlockInput[T]) -> BlockOutput:
         yield "result", input_data.value
 
 
-class InputBlock(InputOutputBlockBase[Any]):
+class InputBlock(Block):
+    """
+    This block is used to provide input to the graph.
+
+    It takes in a value, name, description, default values list and bool to limit selection to default values.
+
+    It Outputs the value passed as input.
+    """
+
+    class Input(BlockSchema):
+        value: Any = SchemaField(description="The value to be passed as input.")
+        name: str = SchemaField(description="The name of the input.")
+        description: str = SchemaField(description="The description of the input.")
+        placeholder_values: List[Any] = SchemaField(
+            description="The placeholder values to be passed as input."
+        )
+        limit_to_placeholder_values: bool = SchemaField(
+            description="Whether to limit the selection to placeholder values.",
```

Comment:
Isn't limiting the placeholder value makes the value essentially enum?
Can we just use enum?